tools/dm/deployment.md

```diff
@@ -0,0 +1,478 @@
+---
+title: DM-Ansible 数据迁移部署方案
+summary: 使用 DM-Ansible 部署 DM 集群
+category: tools
+---
+# 使用 DM-Ansible 部署 DM
```

Comment:
This 1st level heading should be consistent with that name in "title". We can omit the summary in docs-cn.